File: employee.cpp
#include<iostream>
using namespace std;
class employee{
	public:
		int empid;
		char name[30];
		float salary;
		
	public:
		int getinfo(){
			cout<<"Enter ID:";
			cin>>empid;
			cout<<"Enter name:";
			cin>>name;
			cout<<"Enter salary:";
			cin>>salary;
		}
		int displayinfo(){
			cout<<"Id:"<<empid<<endl;
			cout<<"Name:"<<name<<endl;
			cout<<"Salary:"<<salary<<endl;
		}
};
int main(){
	employee emp;
	emp.getinfo();
	emp.displayinfo();
	return 0;
}

File: employee.java
import java.util.Scanner;

public class employee{
    String fn;
    String ln;
    float Msalary;

    void employee(){
        fn="Tejal";
        ln="Powar";
        Msalary=0.0f;
    }

    void getinfo(){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter First name:");
        fn=sc.nextLine();
        System.out.println("enter Last name:");
        ln=sc.nextLine();
        System.out.println("ENter monthly salary:");
        Msalary=sc.nextFloat();
    }

    void setinfo(){
        System.out.println("First name:" +fn);
        System.out.println("Last name:"+ln);
        System.out.println("Monthly salary:"+ Msalary);
    }

    void yearlySalary(){
        System.out.println("Yearly salary of "+fn+" "+ln+" is"+(12*Msalary));
    }
}

// public class EmployeeTest{
    
// }

class Emplo{
    public static void main(String args[]){
        employee emp = new employee();
        // EmployeeTest empTest = new empTest;

        emp.getinfo();
        emp.setinfo();
        emp.yearlySalary();
        // empTest.getinfo();
    }
}
